site/docs/how-to-vote.md

```diff
@@ -0,0 +1,75 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Voting on an Iceberg Release
+
+Before a new version of Apache Iceberg is released, a release candidate will be prepared and a vote
+will occur. Announcements will be made on the Iceberg [developer's mailing list](https://iceberg.apache.org/#community/#mailing-lists)
+and will include links to the following:
+
+- A source tarball
+- A signature (.asc)
+- A checksum (.sha512)
+- Key files
+- GitHub change comparison
+
+After downloading the source tarball, signature, checksum, and key files, follow
+these steps to participate in the voting process.
+
+Import the keys.
+```bash
+gpg --import ${PATH_TO_KEY_FILES}
+```
+
+Verify the signature.
+```bash
+gpg --verify apache-iceberg-${VERSION}.tar.gz.asc
+```
+
+Verify the checksum.
+```bash
+sha512sum -c apache-iceberg-${VERSION}.tar.gz.sha512
+```
+**note**: If you're on a mac that does not have coreutils installed, you can use `shasum -a 512` instead.
+
+Untar the archive and change into the source directory.
+```bash
+tar xzf apache-iceberg-${VERSION}.tar.gz
+cd apache-iceberg-${VERSION}
+```
+
+Run RAT checks to validate license headers.
+```bash
+dev/check-license
+```
+
+Build and test the project using Java 8.
```

Comment:
I don't think we need to say java 8 explicitly, user should check using the java version they prefer.

Reply:
Jack is right. We actually want people to check with other versions, too.

Reply:
Removed the reference to a specific java version.